src/Host/Client/Impl/Host/IRHostConnector.cs

```diff
@@ -1,11 +1,12 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
+using System;
 using System.Threading;
 using System.Threading.Tasks;
 
 namespace Microsoft.R.Host.Client.Host {
-    public interface IRHostConnector {
+    public interface IRHostConnector : IDisposable {
```

Comment:
This entire area (`IRHostConnector`, `IRHostBrokerConnector` and their respective implementations) should be considered a crude prototype for the real thing.